OpenRA.Platforms.Default/Sdl2Input.cs

```diff
@@ -161,7 +161,11 @@ public void PumpInput(IInputHandler inputHandler)
 								Platform.CurrentPlatform == PlatformType.Windows)
 								Game.Exit();
 							else
-								inputHandler.OnKeyInput(keyEvent);
+							{
+								// Only register initial key press
+								if (e.key.repeat == 0)
```

Comment:
I'm not a fan of hardcoding a rather high level implementation detail (backspace in a textfield should repeat) in our lowest level platform code.